src/test/java/org/elasticsearch/search/sort/SimpleSortTests.java

```diff
@@ -1053,6 +1054,7 @@ public void testSortMissingStrings() throws ElasticsearchException, IOException
         flush();
         refresh();
 
+        // TODO: WTF?
```

Comment:
its not lost, its a TODO. i dont know what impact removing it will have on the test. but when i see a sleep like that, I add a wtf.